ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh

```diff
@@ -79,25 +79,19 @@ function dev_part {
 }
 
 function osd_trying_to_determine_scenario {
-  if [[ ! -d /var/lib/ceph/osd || -n "$(find /var/lib/ceph/osd -prune -empty)" ]]; then
-    echo "No bootstrapped OSDs found; trying ceph-disk"
-    osd_disk
-    return
-  fi
-
   if [ -z "${OSD_DEVICE}" ]; then
     echo "Bootstrapped OSD(s) found; using OSD directory"
     osd_directory
     return
```

Comment:
if we are part of the same `if` you don't need to `return`, this is implied by the `if`